Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -69,9 +69,12 @@ void VideoConfig::Load(const std::string& ini_file)
   settings->Get("OverlayStats", &bOverlayStats, false);
   settings->Get("OverlayProjStats", &bOverlayProjStats, false);
   settings->Get("DumpTextures", &bDumpTextures, false);
-  settings->Get("HiresTextures", &bHiresTextures, false);
-  settings->Get("ConvertHiresTextures", &bConvertHiresTextures, false);
-  settings->Get("CacheHiresTextures", &bCacheHiresTextures, false);
+  if (!settings->Get("CustomTextures", &bCustomTextures, false))
+  {
+	  settings->Get("HiresTextures", &bCustomTextures, false);
+  }
```

Comment:
Since the body of this `if` only is one line, you shouldn't use braces.

Also, don't use tabs.